app/views/content/become-a-further-education-teacher.md

```diff
@@ -19,18 +19,16 @@ keywords:
   - apprencticeship
 ---
 
-Further education is any study that takes place after secondary school but outside of higher education (e.g. university).
+If you’re interested in teaching young people and adults over 16 who aren’t studying for a degree, then further education might be for you. It covers three key areas: 
 
-In further education, teaching doesn’t always happen in a classroom. It can take place in community centres, workplaces and private training centres. 
+* vocational courses including plumbing, manufacturing and health care
+* academic classroom based learning of subjects including English and maths, including A-levels in a sixth form college
+* basic skills courses on literacy, numeracy, English for speakers of other languages, and special needs
 
-You could teach a diverse set of learners, including young people and adults.
+Whichever route you choose, you could teach a diverse set of learners, including young people and adults. 
 
-You do not need a degree to start teaching in further education. 
+You don’t always need a prior teaching qualification or an academic degree to start teaching in further education. Many further education training providers offer funded teacher training on the job, so you can start straight away. However, if you’re mainly interested in teaching in a school, you will be best to explore how to train to teach primary and secondary pupils.
```

Comment:
```suggestion
You do not always need a prior teaching qualification or an academic degree to start teaching in further education. Many further education training providers offer funded teacher training on the job, so you can start straight away. However, if you’re mainly interested in teaching in a school, you will be best to explore how to train to teach primary and secondary pupils.
```
Could we add a h2 before this bit, something like 'What qualifications do you need to teach in further education?' to split up the content a bit?